src/components/ui/IconButton/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"; 
+import { View } from "react-native";
+import Icon from "@app/components/ui/Icon";
+import { colors } from "@app/utils";
+
+interface IconButtonProps {
```

Comment:
I think we also need these props for this one:

- color: string
- styles: ViewStyles
- icon: string

Reason is because we'll be using this component for the back button, question button, the location button and the add button